index.bs

```diff
@@ -424,33 +415,38 @@ partial interface Screen /* : EventTarget */ {
 };
 </xmp>
 
-Issue: {{Screen}} needs to derived from {{EventTarget}}. This will require changes to [[CSSOM-VIEW#the-screen-interface]].
+Issue: Make {{Screen}} derive from {{EventTarget}} in [[CSSOM-VIEW#the-screen-interface]].
+
+<!-- ====================================================================== -->
+### {{Screen/isExtended}} attribute ### {#api-screen-isExtended-attribute}
+<!-- ====================================================================== -->
 
 <div algorithm>
 
 The <dfn attribute for=Screen>isExtended</dfn> getter steps are:
 
-1. Return true if the computer system has more than one [=/connected screen=], and false otherwise.
+1. Return true if the device has more than one [=/screen=], and false otherwise.
 
 </div>
 
 <!-- ====================================================================== -->
-### {{Screen}} events ### {#api-screen-events}
+### {{Screen/onchange}} attribute ### {#api-screen-onchange-attribute}
 <!-- ====================================================================== -->
 
 The <dfn attribute for=Screen>onchange</dfn> attribute is an [=/event handler IDL attribute=] whose [=/event handler event type=] is <a event for=Screen>`change`</a>.
 
-When any [=connected screen/basic observable property=] of a {{Window}}'s [=/current screen=] changes, [=/fire an event=] with type <dfn event for=Screen>`change`</dfn> at the {{Window}}'s associated {{Screen}} object.
+When any [=screen/basic observable property=] of the {{Window}}'s [=/current screen=] changes, [=/Queue a global task=] on the [=/relevant global object=] of [=/this=] using the [=/window placement task source=] to [=/fire an event=] with type <dfn event for=Screen>`change`</dfn> at the {{Screen}} object referenced by {{Window}}'s {{Window/screen}} attribute.
```

Comment:
Queuing a task from outside an algorithm triggered by script is a little tricky because there's isn't really a "this" to start with. Notably, you don't say which Window you're talking about. I think it would make sense to write this as: 

```suggestion
When any [=screen/basic observable property=] of a {{Window}} |window|'s [=/current screen=] changes, [=/queue a global task=] on the [=/relevant global object=] of |window| using the [=/window placement task source=] to [=/fire an event=] with type <dfn event for=Screen>`change`</dfn> at the {{Screen}} object referenced by |window|'s {{Window/screen}} attribute.
```

The other language for firing events needs a similar fix. "Queue" also needs to be lower case.

Reply:
Thank you for your extra attention to detail and help here! I'm following up with a separate PR.